Userland/Libraries/LibHTTP/HttpResponse.cpp

```diff
@@ -18,4 +18,63 @@ HttpResponse::~HttpResponse()
 {
 }
 
+StringView HttpResponse::reason_phrase_for_code(int code)
+{
+    VERIFY(code > 0);
+
+    static HashMap<int, String> s_reason_phrases = {
```

Comment:
We're constructing a bunch of `StringView`s, putting them in a `HashMap` of `String`s and convert them back to `StringView`s when returning - Is that actually needed?
Also, can this and the function itself be `constexpr`?

Reply:
It cannot be `constexpr` because `HashMap` does not support it.